Maui: Updated UI in Location category samples

duffh opened the pull request:
# Description

Updated UI for samples in the Location category.

## Type of change

- [x] Bug fix

**Platforms tested on**:

<!--- Delete any that aren't needed -->

- [x] MAUI Windows
- [x] MAUI Android
- [ ] MAUI iOS

## Checklist

- [ ] Runs and compiles on all active platforms
- [ ] Legacy platforms still compile and run (if applicable)
- [ ] Branch is up to date with the latest main/v.next
- [ ] All merge conflicts have been resolved
- [ ] Self-review of changes
- [ ] There are no warnings related to changes
- [ ] No unrelated changes have been made to any other code or project files
- [ ] `sample_sync.py` runs without making changes
- [ ] Codemaid and XAML styler extensions have been run on every changed file
- [ ] Code is commented with correct formatting
- [ ] All variable and method names are good and make sense
- [ ] There is no leftover commented code
- [ ] Screenshots are correct size and display in description tab

nCastle1 reviewed (changes requested):
- I think Location display auto pan modes sample would benefit from a UI update
- The removed button styles causes buttons to look bad on both mac catalyst and Android. The styles between platforms are very inconsistent. I recommend defining a good default style, maybe taking inspiration from Esri's design system. https://developers.arcgis.com/calcite-design-system/components/button/
- Because the width of the NMEA messages is variable, when in landscape layout on Android, the width of the panel keeps changing. There probably needs to be a maximum width defined for the panel when in landscape, or alternatively a maximum width request defined within the sample

ZackAllen commented:
Display device location - `Application.Current.MainPage.DisplayActionSheet` to fix action sheet
NMEA - vertical accuracy text is cropped on desktop. On Android, control panel size rapidly changes with label updates when in horizontal layout

duffh commented:
Thank you for the feedback. I have updated the display auto pan modes UI and addressed the issues with the NMEA text cropping and control panel size. 

I will cover the button style update on a separate PR.
